exporters/otlp/otlptrace/internal/tracetransform/span.go

```diff
@@ -192,7 +192,7 @@ func spanEvents(es []tracesdk.Event) []*tracepb.Span_Event {
 	for i := 0; i < len(es); i++ {
 		events[i] = &tracepb.Span_Event{
 			Name:                   es[i].Name,
-			TimeUnixNano:           uint64(es[i].Time.UnixNano()),
+			TimeUnixNano:           uint64(es[i].Time.UnixNano()), // nolint:gosec // Unix timestamp can't be negative.
```

Comment:
Time prior to Jan 1 1970 (UTC) is represented as negative values in Unix timestamps. We need to bound this prior to the conversion.